pages/managed-mongodb-databases/how-to/connect-database-private-network.mdx

```diff
@@ -0,0 +1,72 @@
+---
+meta:
+  title: How to connect an existing MongoDB® Database Instance to a Private Network
+  description: Instructions for connecting your MongoDB® Database Instance over a Private Network.
+content:
+  h1: How to connect an existing MongoDB® Database Instance to a Private Network
+  paragraph: Instructions for connecting your MongoDB® Database Instance over a Private Network.
+tags: managed-mongodb mongodb private-network database-instance
+dates:
+  validation: 2025-02-17
+  posted: 2025-02-17
+categories:
+  - managed-databases
+  - mongodb
+---
+
+[Private Networks](/vpc/concepts/#private-networks) allows you to enhance the security of your system's architecture by isolating it from the internet.
+
+Using Private Networks improves performance by reducing the latency between your application and your database nodes. It also increases the security of your databases, as Instances in your Private Network can directly communicate with your Database Instance, without passing through the public internet.
+
+You can [create new Database Instances](/managed-mongodb-databases/how-to/create-a-database-instance) to attach to your Private Network, or attach existing ones.
+
+<Macro id="requirements" />
+
+- A Scaleway account logged into the [console](https://console.scaleway.com)
+- [Owner](/iam/concepts/#owner) status or [IAM permissions](/iam/concepts/#permission) allowing you to perform actions in the intended Organization
+- A valid [API key](/iam/how-to/create-api-keys/)
+- A [Managed MongoDB® Database Instance](/managed-mongodb-databases/how-to/create-a-database-instance)
+
+## How to attach a Database Instance to a Private Network
+
+<Message type="note">
+  You can only attach your Database Instance to one Private Network at a time.
+</Message>
+
+1. Click **MongoDB®** under **Managed Databases** on the side menu. A list of your Database Instances displays.
+2. Click the database name or <Icon name="more" /> > **More info** to access the Database Instance information page.
+3. Scroll to the **Network** section.
+4. Click **Attach Private Network** next to **Private endpoint**. A pop-up appears.
+5. Choose one or both of the options below:
+    - Select and attach an existing Private Network and select the network from the drop-down list.
+        <Message type="important">
+          The Database Instance must be attached to a Private Network in the same region.
+        </Message>
+    - Create and attach a new Private Network and enter the name of the network in the form.
+        <Message type="important">
+          If you select this option, a Private Network with default settings will be created in the same region as your Database Instance. Do not use a TLD (e.g. dev, cloud) as a name, to avoid conflicts.
```

Comment:
```suggestion
          If you select this option, a Private Network with default settings will be created in the same region as your Database Instance. Do not use a TLD (e.g. `dev` or `cloud`) as a name, to avoid conflicts.
```